bundles/org.openhab.core.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/config/EnrichedConfigDescriptionParameterDTO.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.io.rest.core.config;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.config.core.ConfigDescriptionParameter.Type;
+import org.eclipse.smarthome.config.core.dto.ConfigDescriptionParameterDTO;
+import org.eclipse.smarthome.config.core.dto.FilterCriteriaDTO;
+import org.eclipse.smarthome.config.core.dto.ParameterOptionDTO;
+
+/**
+ * This is a data transfer object that is used to serialize parameter of an enriched configuration description.
```

Comment:
Could you add a sentence about what this enriched version of the DTO is (what's the difference to the "normal" DTO)?

Reply:
Done.